webapp/utilities/utils.py

```diff
@@ -12,6 +12,35 @@
 from __app__.crop.structure import SensorLocationClass
 
 
+def query_result_to_dict(query_result, date_iso=True):
+    """
+    If we have a single query result, return output as a dict rather than a list
+    Args:
+        query_result: a ResultProxy representing results of the sql alchemy query execution
+    Returns:
+        results_dict: a dict containing the results
+    """
+    if len(query_result) != 1:
+        print("Only call query_result_to_dict if we have a single result.")
```

Comment:
Might turn the `print` calls to `logging.info`, `logging.warning`, or `logging.error`. I think the other source files use those, and they might play more nicely with Azure logging.